index.js

```diff
@@ -156,8 +156,54 @@ var znFilterMatcher = (function() {
 	};
 
 	/**
+	 * Is var an object?
+	 *
+	 * @param	x
+	 * @returns	{Boolean}
+	 * @author	David McNelis <david.mcnelis@wizehive.com>
+	 * @since	1.1.0
+	 */
+	function isObject(x) {
```

Comment:
@dwmcnelis Any reason why `angular.isObject()` was not used?  https://docs.angularjs.org/api/ng/function/angular.isObject

Reply:
Angular is not a dependency. This library can be used in backend services that don't have it.